csug/foreign.stex

```diff
@@ -3093,6 +3093,8 @@ program.
 \cfunction{void}{Sscheme_init}{void (*\var{abnormal}_\var{exit})(void)}
 \cfunction{void}{Sset_verbose}{int \var{v}}
 \cfunction{void}{Sregister_boot_file}{const char *\var{name}}
+\cfunctTwo{void}{Sregister_boot_executable_relative_file}{const char *\var{execpath},}{const char *\var{name}}
+\cfunction{void}{Sregister_boot_relative_file}{const char *\var{name}}
 \cfunction{void}{Sregister_boot_file_fd}{const char *\var{name}, int \var{fd}}
 \cfunctTwo{void}{Sregister_boot_file_fd_segment}{const char *\var{name}, int \var{fd},}{iptr \var{offset}, iptr \var{len}, int \var{close}_\var{after}}
```

Comment:
`Sregister_boot_file_fd_segment()` is actually `Sregister_boot_file_fd_region()` in other places.

And `check_boot_file_state("Sregister_boot_file_fd");` in body of `Sregister_boot_file_fd_region()` might be refined as `check_boot_file_state("Sregister_boot_file_fd_region");`.

Reply:
Thanks! Fixed.